templates/zerver/register.html

```diff
@@ -131,10 +124,9 @@
                 {% endif %}
             </div>
             <div class="help-box margin-top">
+                {{ _("Letters, numbers, and dashes only.") }}
```

Comment:
This is changed again later, right? we can just change it in this commit to "a-z, 0-9, .."